mobify/platform-scaffold
# Remove Spline

Remove the deprecated Spline library to platform-scaffold. Also replace mixins to CSS styles for utility classes. Please look at "Changes" section for all changes.

 **JIRA**: https://mobify.atlassian.net/browse/AMP-36
 **Linked PRs**: N/A

## Changes
- Remove Spline
- Apply new flexbox classes
- Apply new font weight classes 
- Apply new font family classes
- Apply new font size classes
- Use `u-text-uppercase` instead of `u-text-all-caps`class
- Use `u-display-block` instead of `u-block` class
- Remove color and heading-level mixins from lib.
- Add font weight variables
- Add heading variables
- Remove `$light-border-color` variable
- Add `$light-border` variable
- Use `$neutral-00` variable instead of `#fff`
- Extend headings
- Add new utility classes to box-shadow, color, display, layout, text, and z-index 
- Remove position utility 

## How to test-drive this PR
- Remove node_modules folder
- `npm install`
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Check Merlins Home page, PDP, and PLP to make sure it looks fine.
